engineer.atlas.Nyxt.yaml

```diff
@@ -0,0 +1,83 @@
+app-id: engineer.atlas.Nyxt
+# TODO Should be replaced it with org.gnome.Platform.  Shared libraries aren't
+# available from org.gnome.Platform, therefore we use the development runtime.
+runtime: org.gnome.Sdk
```

Comment:
```suggestion
runtime: org.gnome.Platform
```

Reply:
this doesn't explain anything.

shared libraries ARE there because that's the default.